src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,59 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0504: r##"
+This error occurs when an attempt is made to move a borrowed variable into a
+closure. For example:
+
+```compile_fail
```

Comment:
I just hope it won't get in the way for readers. What do you think about this @steveklabnik?